flair/embeddings.py

```diff
@@ -16,6 +16,8 @@
 from pytorch_transformers import (
     BertTokenizer,
     BertModel,
+    DistilBertTokenizer,
```

Comment:
Perhaps move these declarations to the `if bert_model_or_path.startswith("distilbert"): ` part of the BertEmbeddings?

This way, it won't trigger an error in Travis when initializing the embeddings classes.

Reply:
Thanks :) I changed the import logic. Now an error message is thrown whenever `DistilBert*` is not found!